### PR TITLE
Update documentation for LabelPicker #1315

### LINKED ISSUE #1315: Update documentation for Label Picker
TODO
- [x] query field will be disabled after mouse click
- [x] emphasis how label matching works

### PR DESCRIPTION
**Pull request by jinified**
Fixes #1315 

## Changes
- added warning about disabled text field after mouse click

### CODE REVIEW

**Comment by jinified** [comment]
@dariusf  ready for review

**Comment by dariusf** [comment]
- We can remove the old demo image I think, it's in the history anyway.
- The typing part can be moved up above the clicking part, since that's the more common use case we want to support.
- The part on space toggling the label can be explained in a bit more detail. Maybe say that pressing space commits the choice, and the label will be applied if it is not present, or removed if it is?

File: docs/labels.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ Pressing <kbd>L</kbd> after selecting an issue card brings up the Label Picker.
 
 The Label Picker is split into 2 parts, the part above the text field shows the labels that are currently applied to the particular item. The part below the text field shows all the labels that current exist in the repository. 
 
-Clicking on labels in either part toggles their state. 
+Clicking on labels in either part toggles their state.  
+> Text field will be disabled after clicking on a label.
```
Comment (dariusf) [inline docs/labels.md]: Hmm, this quoted style isn't used in the rest of the document. We can just include this as part of the previous line.

File: docs/labels.md [hunk 2]
```diff
@@ -19,15 +19,16 @@ Pressing <kbd>L</kbd> after selecting an issue card brings up the Label Picker.
...
 ![](images/labels/main.png?raw=true)
 
-The Label Picker is split into 2 parts, the part above the text field shows the labels that are currently applied to the particular item. The part below the text field shows all the labels that current exist in the repository. 
+Typing in the text field highlights the labels that contain the query. 
 
-Clicking on labels in either part toggles their state. 
+![](images/labels/highlight.png?raw=true)
 
-![](images/labels/demo.gif?raw=true)
+The Label Picker is split into 2 parts, the part above the text field shows the labels that are currently applied to the particular item. The part below the text field shows all the labels that current exist in the repository.
```
Comment (damithc) [inline docs/labels.md]: `currently exist`?

File: docs/labels.md [hunk 3]
```diff
@@ -39,4 +40,4 @@ There is support for label groups, typing `p.l` highlights `priority.low` and so
 
 Finally, press `Confirm` or <kbd>Enter</kbd> to accept the label selection or press `Cancel` or <kbd>Escape</kbd> to cancel and close the Label Picker. 
 
-Label changes are not sent to GitHub immediately to allow the user a short window of time to undo the changes. The `Undo` button on the notification reverts the label changes and pressing <kbd>Ctrl</kbd>+<kbd>z</kbd> will do the same. 
\ No newline at end of file
+Label changes are not sent to GitHub immediately to allow the user a short window of time to undo the changes. The `Undo` button on the notification reverts the label changes and pressing <kbd>Ctrl</kbd>+<kbd>z</kbd> will do the same.
```
Comment (damithc) [inline docs/labels.md]: What changed here?

**Comment by damithc** [comment]
Some minor changes requested.

File: docs/labels.md (hunk 3 above)
Comment (jinified) [inline docs/labels.md]: Technically nothing. I am not sure why is highlighted as a change.

**Pull request merged**